daemon/src/lib.rs

```diff
@@ -13,7 +16,18 @@ pub async fn process(socket: TcpStream) {
         .expect("invalid format from client");
     let message: Message = serde_json::from_reader(bytes.as_ref()).expect("failed to read");
     match message {
-        Message::Dummy => println!("Got a Dummy message!"),
+        Message::FindInformationOn(search) => {
+            println!("Got a directive to find information on \"{}\"", search);
+            let res = make_request(
+                search,
+                Target::StackExchange(Site::Site(String::from("stackoverflow"))),
```

Comment:
I think this needs to eventually come from a `HashMap<&str, Target>` that is built from an external data source so that this set of capabilities isn't fully fixed.

Reply:
So it might make sense to actually pull this from a `Lazy<HashMap<&str, Target>>` using the `Lazy` struct from [once_cell](https://docs.rs/once_cell/1.4.0/once_cell/).

Reply:
I'm not quite following yet. What are we storing in the HasMap? i.e. what are those keys conceptually?

I woke up wondering how hard it would be to teach 7roxy Rust. . . and then build a web explorer. . . and then have it generate Rust code that could connect up to arbitrary data sources by parsing the API documentation.

And since I'm thinking of having 7roxy open PR's to itself. . . I'm wondering if we should punt on the HashMap generally (depending on what it is obviously haha). Because in the world I'm imagining. . . I would want to review a PR from 7roxy that is a brand new file (representing the configuration of the data source) and then adding an additional permutation to some Enum type.

Reply:
I have extracted: https://github.com/7hoenix/7roxy/issues/9 to represent this work.